### DISCUSSION TITLE
Data is not reflected in branch

### DISCUSSION THREAD

**Discussion by leowmjw** [planetscale/discussion#45] [Bug Reports] [answered]
Data was inserted manually direct into the main branch via pscale shell session before a branch is taken of the main.

When a branch is made from the main; I would expect the existing data to be cloned over as well.
However, this does not seem to be the case; see below for steps to reproduce:

```
$ pscale shell temporal main
temporal/main> SELECT * FROM namespace_metadata;
+--------------+----------------------+
| partition_id | notification_version |
+--------------+----------------------+
|        54321 |                    1 |
+--------------+----------------------+
temporal/main> exit
```

... Create new branch **dev** ...

```
$ pscale shell temporal dev
temporal/dev> SELECT * FROM namespace_metadata;

```

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @leowmjw,

Currently branches include the schema only. We're going to make data branching available in the future.
